src/server_manager/web_app/gcp_account.ts

```diff
@@ -21,6 +21,13 @@ import * as server from '../model/server';
 
 import {GcpServer} from './gcp_server';
 
+/** Returns a unique, RFC1035-style name as required by GCE. */
+function makeInstanceName(): string {
```

Comment:
Perhaps rename to `makeInstanceGcpName` to make is clearer that that's the GCP name, not the application name.